lms/djangoapps/verify_student/views.py

```diff
@@ -1154,10 +1154,22 @@ def results_callback(request):
     except SoftwareSecurePhotoVerification.DoesNotExist:
         log.error("Software Secure posted back for receipt_id %s, but not found", receipt_id)
         return HttpResponseBadRequest("edX ID {} not found".format(receipt_id))
+    context = {'user': attempt.user}
+    email_template_context = {'platform_name': settings.PLATFORM_NAME}
     if result == "PASS":
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
+        expiry_date = datetime.date.today() + datetime.timedelta(days=365)
+        expiry_date = "{}/{}/{}".format(expiry_date.month, expiry_date.day, expiry_date.year)
+        email_template_context['full_name'] = context['user'].profile.name
+        email_template_context['expiry_date'] = expiry_date
+        context['email_template_context'] = email_template_context
+        context['subject'] = _("Your {platform_name} ID Verification Approved").format(
+            platform_name = settings.PLATFORM_NAME
+        )
+        context['message'] = 'emails/successfull_verification_email.txt'
+        send_verification_status_email(context)
```

Comment:
are we only sending verification status emails for success? if so, shouldn't we define all the context variables for the email in this if block?

Reply:
actually this is just a little sub-task.I need to send email of different statuses of the user.
https://openedx.atlassian.net/browse/LEARNER-617